src/modules/Accordion/Accordion.js

```diff
@@ -84,22 +94,39 @@ export default class Accordion extends Component {
     // The default prop should always win on first render.
     // This default check should then be removed.
     if (typeof this.props.defaultActiveIndex === 'undefined') {
-      this.trySetState({ activeIndex: -1 })
+      this.props.exclusive ? this.trySetState({ activeIndex: -1 }) : this.trySetState({ activeIndex: [-1] })
     }
+
+    //TODO: Can not accept an activeIndex and be inclusive at the same time
+    // if(this.props.exclusive == false && _.isNumber(this.props.activeIndex)){
+    //   this.trySetState({ activeIndex: [ this.props.activeIndex ] })
+    // }
```

Comment:
So if you I understand correctly `defaultActiveIndex` should be allowed to an array from props?